packages/site/src/OffRamp.tsx

```diff
@@ -130,39 +130,50 @@ export const OffRamp: FunctionComponent<OffRampProps> = (
-              Step 3: Send your funds from selected address or just enter amount and sign transaction. Please note that
-              transfers from other addresses could cause unrecoverable loss of assets.
+              Step 3: Just enter amount and confirm transaction with MetaMask.
             </CardDescription>
           </CardHeader>
           <CardContent className="grid gap-4">
-            <div className="w-full max-w-sm items-center">
-              <Label htmlFor="address">Address</Label>
-              <Input type="text" id="address" placeholder="address" readOnly={true} value={selectedAsset.offRamp?.address}/>
-            </div>
-            {<>
               <div className="w-full max-w-sm items-center">
                 <Label htmlFor="amount">Amount {selectedAsset!.asset?.shortName}</Label>
-                <Input type="text" id="amount" placeholder={`amount in ${selectedAsset!.asset?.shortName}`} value={amount}  onChange={handleAmountChange} disabled/>
+                <Input type="text" id="amount" placeholder={`amount in ${selectedAsset!.asset?.shortName}`}
+                       value={amount} onChange={handleAmountChange} disabled={false}/>
               </div>
+              <div className="w-full max-w-sm items-center">
+                <BankAccountComponent account={getOffRampBankAccount(account)}/>
+              </div>1§
               <div>
-                <Button className="w-full" onClick={handleTransfer} disabled>Sign and submit transaction</Button>
+                <Button className="w-full" onClick={handleTransfer} disabled={false}>
+                  <img src={Metamask} className="mr-2 h-4 w-4"/>
+                  Send with MetaMask
+                </Button>
               </div>
-            </>}
           </CardContent>
           <CardFooter className="grid gap-4">
+            <div className="relative m-2">
+              <div className="absolute inset-0 flex items-center"><span className="w-full border-t"></span></div>
+              <div className="relative flex justify-center text-xs"><span
+                className="bg-background px-2 text-muted-foreground">or</span></div>
+            </div>
+
             <div className=" flex items-center space-x-4 rounded-md border p-4">
-              <PiggyBankIcon/>
+              <WalletIcon/>
               <div className="flex-1 space-y-1">
-                {false &&<p className="text-sm font-medium leading-none">
+                {false && <p className="text-sm font-medium leading-none">
                   Push Notifications
                 </p>}
                 <p className="text-sm text-muted-foreground">
-                  You will receive your funds directly to your bank account.
+                  Alternatively send {selectedAsset!.asset?.shortName} from the <u>selected address</u> to the <u>following
+                  address</u>.
+                  Please note that transfers from other addresses could cause unrecoverable loss of assets.
```

Comment:
```suggestion
                  Please note that transfers from other addresses will be bounced back, minus network fees.
```
That's extreme... maybe check this wording with James.